src/pepr/operator/controllers/network/allow-ingress-sidecar-monitoring.ts

```diff
@@ -0,0 +1,40 @@
+import { kind } from "pepr";
+
+export function allowIngressSidecarMonitoring(namespace: string): kind.NetworkPolicy {
```

Comment:
Probably not for this PR, but it would be awesome if we could converge on a standard way of creating NetPols. Here we use the JSON payload, but in the platform components (ie. Neuvector, etc,) we use that custom `uds-package.yaml`

Reply:
Really interesting point, I wrote these netpols when we didnt have the generator yet, moving to the generator for these as well makes a lot of sense and clearly demonstrates the difference between how UDS Operator handles these rules and how K8s does. 

Note we can't "directly use the UDS Package CRD", because these are actually processes running by the operator in response to a CR.

![Screenshot 2024-01-04 at 10 37 30 PM](https://github.com/defenseunicorns/uds-core/assets/882485/8b72eab5-c9fe-4f0c-84b0-4eec3a26cd7e)
![Screenshot 2024-01-04 at 10 37 22 PM](https://github.com/defenseunicorns/uds-core/assets/882485/259f2774-a8fb-4420-8278-5d6beafbd8bd)